src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/params.py

```diff
@@ -635,3 +652,28 @@ def _configure_security_policy_storage_params(cmd):
                options_list=('--end-ip-address',),
                help='The end IP address of the firewall rule. Must be IPv4 format. Use value'
                ' \'0.0.0.0\' to represent all Azure-internal IP addresses.')
+
+#####
+#           sql server key
+#####
+
+with ParametersContext(command='sql server key') as c:
+    c.argument('server_name', arg_type=server_param_type)
+    c.argument('key_name', options_list=('--name', '-n'))
+    c.argument('key_id', options_list=('--key-id', '-k'),
+               required=True,
+               help='The Azure Key Vault keyid of the server key')
```

Comment:
👍

Reply:
Is "keyid" standard terminology that is also used elsewhere?

Reply:
We use "keyid" in the corresponding powershell cmdlet. Azure Key Vault uses "kid" or "Key Identifier". I'm considering switching this to "kid" to be consistent with AKV, but I'm concerned that "kid" will be confusing outside the context of Key Vault. I'll discuss with my team.

Reply:
Discussed with my team... it's best to be consistent with AKV here. I'll modify this to be "kid".